Yaohan_National_Parks/css/style.css

```diff
@@ -0,0 +1,231 @@
+/* stylelint-disable font-family-name-quotes */
+body {
+  margin: 0;
+  border-width: 0;
+  box-sizing: border-box;
+  height: 100vh;
+
+  font-family: "Roboto", sans-serif;
+}
+
+/* Cover page
+Some of this chunk of code is referenced from ChatGPT.
+*/
+#cover-page {
+  position: relative;
+  width: 100vw;
```

Comment:
**Issue:** Currently, your front page has a horizontal scroll bar (in Chrome), which I assume is not what you want. If you want to remove it, just delete this line. Mjumbe and I are not sure why this can solve the issue, but it is what it is.

```suggestion
```